cfme/tests/intelligence/reports/test_menus.py

```diff
@@ -105,9 +133,10 @@ def test_shuffle_first_level(appliance, group, report_menus):
 @pytest.mark.tier(3)
 @pytest.mark.parametrize("group", GROUPS)
 @test_requirements.report
-def test_add_reports_to_available_reports_menu(appliance, request, group,
-                                               report_menus, custom_report_values):
-    """This test case moves custom menu to existing menus
+def test_add_reports_to_available_reports_menu(
+    appliance, request, group, report_menus, get_custom_report
+):
+    """This test case moves custom report to existing menus
```

Comment:
Remove ```request```

Reply:
`get_custom_report` uses a fixture that requires request.